src/ds/cdllist.h

```diff
@@ -11,7 +11,7 @@ namespace snmalloc
    * Special class for cyclic doubly linked non-empty linked list
    *
    * This code assumes there is always one element in the list. The client
-   * must ensure there is a sentinal element.
+   */ust ensure there is a sentinal element.
    **/
```

Comment:
Not GitHub is highlighting them as bad!

Reply:
Don't know how that happened. I just performed a find and replace.  Reverting and doing the same worked.